library/globals.inc.php

```diff
@@ -1527,6 +1538,7 @@
       xl('Default Calendar View'),
       array(
        'providerAgenda' => xl('Agenda'),
+       'providerAgenda2Day' => xl('Agenda 2 Day'),
```

Comment:
what does this mean?  Do you mean "Display 2 Days in Agenda View"

Reply:
Yes